CMSgov/onemac
# OY2 20599: Update Waiver Authority for App K Amendment Details

Story: https://qmacbis.atlassian.net/browse/OY2-20599
Endpoint: See github-actions bot comment

### Details
Appendix K's have a different Waiver Authority from 1915(b) waivers and it is always 1915(c) HCBS

### Changes
- Updated the DetailView niceLabel to the new text
- set that value as the default for Waiver Authority being null (currently, the Appendix K form does not set a waiver authority, so app k  items in the database have no waiverAuthority (the seed data does, however))
- added the default 1915(c) value to the waiverAuthorities "choices" as the single choice
- modified the form to handle only one waiverAuthority choice as something to send straight through

### Test Plan
1. Login as a user who can view packages
2. Navigate to Package Dashboard, Waivers Tab
3. Locate an Appendix K Amendment in the table
4. Click the ID to view details
5. Verify the Waiver Authority follows the story AC
